setup.py

```diff
@@ -30,7 +30,10 @@
         'tabulate'
     ],
     url='https://github.com/dataxu/github-pr',
-    version='1.1.1',
+    version='2.0.0',
     scripts=['github_pr.py'],
     keywords=['github'],
+    dependency_links=[
+        "git+https://github.com/dataxu/PyGithub#egg=PyGithub-1.26.1"
```

Comment:
is this necessary? What does it do that is not accomplished by `install_requires`. This might cause problems if a user of this library requires a newer version of `PyGithub`